src/ert/scheduler/driver.py

```diff
@@ -0,0 +1,89 @@
+import asyncio
+import os
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import (
+    MutableMapping,
+    Optional,
+    Tuple,
+)
+
+
+class JobEvent(Enum):
+    STARTED = 0
+    COMPLETED = 1
+    FAILED = 2
+    ABORTED = 3
+
+
+class Driver(ABC):
+    """Adapter for the HPC cluster."""
+
+    event_queue: asyncio.Queue[Tuple[int, JobEvent]]
```

Comment:
We need to do typing.Queue here:
```from typing import Queue
event_queue: Queue[Tuple[int, JobEvent]] ....
```

Reply:
I think the solution is `from __future__ import annotations`

Reply:
Or that :)